### PR TITLE
Fix possible snmpd crash when sub agent timeout.

### PR DESCRIPTION
**Pull request by renukamanavalan**
Upon snmpd closes a netsnmp_agent_session due to snmp_timeout
there is a possibility of crash due to stale memory access.

This is a patch from source-forge:net-snmp. commit-id #793d59

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/snmpd/patch-5.7.3+dfsg/0006-From-Jiri-Cervenka-snmpd-Fixed-agentx-crashing-and-or-freezing-on-timeout.patch [hunk 1]
```diff
@@ -0,0 +1,211 @@
+From a5782d0673044ad0c621daed7975f53238bb038e Mon Sep 17 00:00:00 2001
+From: Renuka Manavalan <remanava@microsoft.com>
+Date: Tue, 10 Sep 2019 17:51:45 +0000
+Subject: [PATCH] Patch from SourceForge: net-snmp commit #793d59 Avoids snmpd
+ crash when sub agent timesout.
+
+---
```
Comment (lguohan) [inline src/snmpd/patch-5.7.3+dfsg/0006-From-Jiri-Cervenka-snmpd-Fixed-agentx-crashing-and-or-freezing-on-timeout.patch]: can you put this original commit id in this patch?

**Comment by renukamanavalan** [comment]
retest this vs please.

**Comment by renukamanavalan** [comment]
retest vsimsge please.

**Comment by renukamanavalan** [comment]
retest vs please.

**Comment by renukamanavalan** [comment]
retest this please.

**Comment by renukamanavalan** [comment]
retest this please

**Pull request merged**